analyzers/tests/SonarAnalyzer.UnitTest/TestCases/WeakSslTlsProtocols.cs

```diff
@@ -122,8 +122,7 @@ public void SslProtocolsNonComplaint()
 
             var protocol = SslProtocols.None;
 
-            // FP - the following cases should be valid
-            bool isSafe = protocol == SslProtocols.Tls ? true : false; // Noncompliant
+            bool isSafe = protocol == SslProtocols.Tls ? true : false;
```

Comment:
Maybe this could be moved to SslProtocolsCompliant method. And as we now do not raise for whilestatements. We should add a case for that too.